### ISSUE TITLE
Choose a Linter for Accessibility

### ISSUE THREAD

**Issue by sabdul14** [FordLabs/PeopleMover#388]
# Feature Request

**As a** FLIP Team member
**I would** like to know which linter is best suited for our accessibility needs
**so that** I am made aware of code that does not meet accessibility standards

## Acceptance Criteria

Acceptance of this story is research, agreement, & communication to team on most appropriate linter for our needs.

## Notes

One you could look into that was discussed during Accessibility Audit Findings meeting was: 
https://www.npmjs.com/package/eslint-plugin-jsx-a11y

**Comment by michaelrygiel** [comment]
We should definitely use jest-axe to write our own accessibility tests since [~30% of issues are found by automated testing.](https://accessibility.blog.gov.uk/2017/02/24/what-we-found-when-we-tested-tools-on-the-worlds-least-accessible-webpage/)
